tests/acceptance/features/apiSpaces/setQuota.feature

```diff
@@ -166,3 +166,39 @@ Feature: Set quota
       | Space Admin |
       | User        |
       | User Light  |
+
+  @issue-9245
+  Scenario: user tries to set quota greater than 885GB
+    Given the administrator has assigned the role "Space Admin" to user "Alice" using the Graph API
```

Comment:
we should add it in this scenario
https://github.com/owncloud/ocis/blob/1d2a868fba6c5333381533d8d3a577b8c8c8e36f/tests/acceptance/features/apiSpaces/quota.feature#L22-L26